firestore/client/firestore_client.bzl

```diff
@@ -0,0 +1,9 @@
+# DO NOT EDIT -- GENERATED CODE
```

Comment:
Please remove this file from this PR.

Reply:
I did not get why to create separate PR for the bzl files.
What should be the title for the new PR for bzl files.

Reply:
All Build looking as successful in last commit, which build was broken.